sidekick/utils.py

```diff
@@ -22,7 +22,7 @@ def get_current_week_number():
 
 
 def clean_message(message):
-    return " ".join(message.replace("\n", " ").replace("\t", " ").split())
+    return " ".join(message.replace("\n", " ").replace("\t", " ").split()) or " "
```

Comment:
`.split()` should split on all whitespace by default, including `\n` and `\t`, so we can probably simplify this to `" ".join(message.split()) or " "`, or use regex `re.sub("\W+", " ", message)`

Reply:
wow, let me try.